modules/oxide/src/less/theme/components/edit-area/edit-area.less

```diff
@@ -4,13 +4,25 @@
 
 @edit-area-iframe-background-color: @color-white;
 @edit-area-border: 0;
+@edit-area-border-color: #2D6ADF;
 
 .tox {
   .tox-edit-area {
     display: flex;
     flex: 1;
     overflow: hidden;
     position: relative; // Required for the iframe to resize in IE and Edge
+    &::before {
+      border: 2px solid @edit-area-border-color;
+      border-radius: 4px;
+      content: '';
+      inset: 2px;
+      opacity: 0;
+      pointer-events: none;
+      position: absolute;
+      transition: opacity .15s;
+      z-index: 10;
```

Comment:
Why was the z-index needed here?

Reply:
It makes the border visible by bringing the pseudo element forward. The value can be 1 though